packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -148,6 +169,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		tabIndex: 0,
 		role: 'document',
 		'aria-label': blockLabel,
+		'aria-description': blockListSettings ? blockDescription : undefined,
```

Comment:
It looks like `blockListSettings` is being used to determine whether the block supports child blocks here.

I wonder if it's worth adding a `hasInnerBlockList` selector function to the block editor store. It could be used here to make this code a bit more readable. It would also make future refactoring of code easier if the way existence of a block list is determined changes.

From memory, I think @youknowriad has also run into this before, and there might be some pitfalls I'm unaware of.

Reply:
There's a difference between: my block is a container block and my block has a non empty list of inner blocks. I guess we might need both. For the latter `!! getBlock( clientId )?.innerBlocks?.length` might be sufficient.